operate/client/src/modules/components/StateOverlay/index.tsx

```diff
@@ -42,6 +40,11 @@ const StateOverlay: React.FC<Props> = observer(
   }) => {
     const showStatistic = count !== undefined;
 
+    // FIXME: What icons and colors should be used for these new states?
```

Comment:
Problem relates to https://github.com/camunda/camunda/issues/27347#issuecomment-3317590945, which has been identified in https://github.com/camunda/camunda/pull/38542. This `StateOverlay` additionally adds a color to the question.